src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -24,23 +28,38 @@ public class ShooterSubsystem extends SubsystemBase {
             ShooterConstants.kA);
 
     public ShooterSubsystem() {
-        flywheel1.setInverted(true);
+        ShooterConstants.configureTable();
 
         flywheel1.setSmartCurrentLimit(Constants.CURRENT_LIMIT);
         flywheel2.setSmartCurrentLimit(Constants.CURRENT_LIMIT);
 
+        flywheel1.getPIDController().setP(ShooterConstants.kP);
+        flywheel2.getPIDController().setP(ShooterConstants.kP);
+
+        flywheel1.getPIDController().setFF(0.00016);
+        flywheel2.getPIDController().setFF(0.00016);
+
         // flywheel1.setOpenLoopRampRate(.2);
         // flywheel2.setOpenLoopRampRate(.2);
 
         flywheel1.setIdleMode(IdleMode.kCoast);
         flywheel2.setIdleMode(IdleMode.kCoast);
 
+        flywheel1.setInverted(true);
         flywheel2.follow(flywheel1, true);
     }
 
-    public void rampUpShooter(double speed) {
-        flywheel1.set(speed);
-        flywheel2.set(speed);
+    public void setRPM(double speed) {
+        SmartDashboard.putNumber("Flywheel Set Point", speed);
+        flywheel1.getPIDController().setReference(speed, ControlType.kVelocity);
+        flywheel2.getPIDController().setReference(speed, ControlType.kVelocity);
+    }
+
+    public void setRPM(){
+        double speed = encoder.getVelocity() + 200; 
```

Comment:
wouldn't this just bump the RPM continuously if held? It looks like this isn't called at the moment in RobotContainer? How does this work sorry if I didn't understand

Reply:
Sorry, this was for debugging and I forgot to remove it before merging.